### ISSUE TITLE
MWE

### ISSUE THREAD

**Issue by yegor256** [yegor256/jpeek#17]
Let's implement MWE cohesion metric.

Y. Liu, D. Poshyvanyk, R. Ferenc, T. Gyim´othy, and N. Chrisochoides,
“Modeling class cohesion as mixtures of latent topics,”
in Software Maintenance, 2009. ICSM 2009. IEEE International
Conference on. IEEE, 2009, pp. 233–242.

PDF is not available :(

http://ieeexplore.ieee.org/document/5306318/

**Comment by 0crat** [comment]
Job [#17](https://github.com/yegor256/jpeek/issues/17) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://datum.zerocracy.com/pages/policy.html#29): +15 points just awarded to [@yegor256](http://www.0crat.com/u/yegor256), total is [+7225](http://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#17](https://github.com/yegor256/jpeek/issues/17) assigned to @skapral[/z](https://www.0crat.com/u/skapral). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by skapral** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/jpeek/issues/17#issuecomment-360608748))

@skapral This is what I know about this job, as in [§32](http://datum.zerocracy.com/pages/policy.html#32):

  * The job [#17](https://github.com/yegor256/jpeek/issues/17) is in scope for 11days
  * The role is `DEV`
  * The job is assigned to @skapral[/z](https://www.0crat.com/u/skapral) for 2days
  * These users are banned: @yegor256[/z](https://www.0crat.com/u/yegor256): This user reported the ticket

**Comment by skapral** [comment]
@0crat waiting #110

**Comment by 0crat** [comment]
> @0crat waiting #110 ([here](https://github.com/yegor256/jpeek/issues/17#issuecomment-360816393))

@skapral The impediment for [#17](https://github.com/yegor256/jpeek/issues/17) was registered successfully by @skapral[/z](https://www.0crat.com/u/skapral)

**Comment by skapral** [comment]
@0crat status

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
java.lang.IllegalStateException: thread 'BkParallel-5' in group 'main' interrupted
	at com.jcabi.aspects.aj.MethodLogger.wrap(MethodLogger.java:184)
	at com.jcabi.aspects.aj.MethodLogger.ajc$inlineAccessMethod$com_jcabi_aspects_aj_MethodLogger$com_jcabi_aspects_aj_MethodLogger$wrap(MethodLogger.java:1)
	at com.jcabi.aspects.aj.MethodLogger.wrapClass(MethodLogger.java:140)
	at com.jcabi.github.Comment$Smart.issue(Comment.java:181)
	at com.zerocracy.radars.github.ReSafe.lambda$react$1(ReSafe.java:70)
	at org.cactoos.func.FuncOf.lambda$new$3(FuncOf.java:88)
	at org.cactoos.func.FuncOf.apply(FuncOf.java:104)
	at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:90)
	at org.cactoos.func.IoCheckedFunc.lambda$apply$0(IoCheckedFunc.java:60)
	at org.cactoos.func.IoCheckedBiFunc.apply(IoCheckedBiFunc.java:66)
	at org.cactoos.func.IoCheckedFunc.apply(IoCheckedFunc.java:61)
	at com.zerocracy.radars.github.ReSafe.react(ReSafe.java:80)
	at com.zerocracy.radars.github.ReOnComment.send(...
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm/tree/1.0-SNAPSHOT) /cc @yegor256

**Comment by skapral** [comment]
@0crat status

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/yegor256/jpeek/issues/17#issuecomment-361078847). You must always start your message with my name if you want to address it to me, see [§1](http://datum.zerocracy.com/pages/policy.html#1).

**Comment by skapral** [comment]
@0crat status

**Comment by 0crat** [comment]
@skapral[/z](https://www.0crat.com/u/skapral) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by skapral** [comment]
@0crat status

**Comment by 0crat** [comment]
Are you speaking to me or about me [here](https://github.com/yegor256/jpeek/issues/17#issuecomment-361664410). You must always start your message with my name if you want to address it to me, see [§1](http://datum.zerocracy.com/pages/policy.html#1).

**Comment by 0crat** [comment]
@skapral[/z](https://www.0crat.com/u/skapral) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by skapral** [comment]
@yegor256 please close this issue since PR was merged.

**Comment by yegor256** [comment]
@skapral thanks!

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @skapral[/z](https://www.0crat.com/u/skapral), total is [+165](https://www.0crat.com/u/skapral)

**Comment by 0crat** [comment]
The job [#17](https://github.com/yegor256/jpeek/issues/17) is now out of scope

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#143](https://github.com/yegor256/jpeek/issues/143), [#144](https://github.com/yegor256/jpeek/issues/144) are still not solved; solved: [#145](https://github.com/yegor256/jpeek/issues/145).

**Comment by 0pdd** [comment]
@yegor256 all 3 puzzles are solved here: [#143](https://github.com/yegor256/jpeek/issues/143), [#144](https://github.com/yegor256/jpeek/issues/144), [#145](https://github.com/yegor256/jpeek/issues/145).
